cli/tests/integrations/test_analytics.py

```diff
@@ -115,64 +173,6 @@ def test_config_reporting_false():
         assert requests.post.call_count == 0
 
 
-@_mock
-def test_production_setting_true():
```

Comment:
When this failure happened (tests failed on py3.4 but works fine for py2.7), we noticed that tests have different order for py2.7 and py3.4(py3.4 runs tests in same order as they are ordered in python module). So we tried to change tests order (as they runned by py2.7). And this helped. Also we try to use py.test annotations like `@pytest.mark.first` but they doesn't work. As I mentioned here https://github.com/mesosphere/dcos-cli/pull/218#discussion_r32337010, we doesn't know exact reason of this failure, but if needed we can research this problem (we also tried, but without any success).